kotlinsample/src/main/java/com/airbnb/epoxy/kotlinsample/helpers/KotlinViewBindingModel.kt

```diff
@@ -0,0 +1,37 @@
+package com.airbnb.epoxy.kotlinsample.helpers
+
+import android.view.View
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.viewbinding.ViewBinding
+import com.airbnb.epoxy.EpoxyModel
+import kotlin.reflect.KFunction1
+
+abstract class KotlinViewBindingModel<T : ViewBinding>(
+    @LayoutRes private val layoutRes: Int,
+    private val bindFunction: KFunction1<@ParameterName(name = "view") View, T>
+) : EpoxyModel<View>() {
+
+    private var _binding: T? = null
+
+    protected val binding: T
+        get() = _binding ?: error("Accessing not bound ViewBinding.")
+
+    abstract fun bind()
+
+    override fun bind(view: View) {
+        _binding = bindFunction(view)
```

Comment:
bind can be called multiple times when data is updated. I think this only needs to be done if `_binding` is null. `_binding = _binding ?: bindFunction(view)`